--- trunk/intltool-update.in.in	2007/02/14 04:12:19	671
+++ trunk/intltool-update.in.in	2007/03/02 16:43:19	674
@@ -563,13 +563,21 @@
     exit 1;
 }
 
+sub isIntltoolExtractInPath
+{
+    my ($file) = @_;
+    # If either a file exists, or when run it returns 0 exit status
+    return 1 if ((-x $file) or (system("$file >/dev/null") == 0));
+    return 0;
+}
+
 sub GenerateHeaders
 {
     my $EXTRACT = $ENV{"INTLTOOL_EXTRACT"} || "intltool-extract";
 
     ## Generate the .h header files, so we can allow glade and
     ## xml translation support
-    if (! -x "$EXTRACT")
+    if (! isIntltoolExtractInPath("$EXTRACT"))
     {
 	print STDERR "\n *** The intltool-extract script wasn't found!"
 	     ."\n *** Without it, intltool-update can not generate files.\n";
